tests/hash-graph-test-data/rust/src/entity/book.json

```diff
@@ -1,5 +1,7 @@
 {
-  "https://blockprotocol.org/@alice/types/property-type/name/": "The Time Machine",
+  "https://blockprotocol.org/@alice/types/property-type/name/": [
+    "The Time Machine"
+  ],
   "https://blockprotocol.org/@alice/types/property-type/blurb/": "brulb",
```

Comment:
Sorry, this is not important, but... `brulb`?! 😂

Reply:
I thought that was on purpose and I always left it as it is just because ... why not? 😂

Reply:
I think @nonparibus mainly flagged `"brulb"` instead of `"blurb"` 😄

Reply:
^ Which is actually `blurb` reversed, so I guess that _is_ on purpose.